newsroom/agenda/model.py

```diff
@@ -0,0 +1,15 @@
+from datetime import datetime
+
+from superdesk.core.resources import ResourceModel, validators
+from typing_extensions import Annotated
+
+
+class FeaturedResourceModel(ResourceModel):
+    _id: Annotated[
```

Comment:
We have the [id](https://github.com/superdesk/superdesk-core/blob/async/superdesk/core/resources/model.py#L69) field defined on the parent ResourceModel class, use that field here instead of `_id`. To make sure the field is a string, you can redefine it here (without the validation, as MongoDB should handle that for us already).
```python
id: Annotated[str, Field(alias="_id")]
```

On a side note, fields starting with an `_` are reserved for private fields. To get around this, we use Pydantic's alias feature, as shown above.